cmf-cli/resources/template_feed/init/EnvironmentConfigs/%EnvironmentName%.yml

```diff
@@ -5,6 +5,9 @@ variables:
   # Run Settings
   RunSettings: 'integration.runsettings'
 
+  # Location of release candidate packages being tested in a release
+  CandidatePackages: '<%= $CLI_PARAM_RCRepo %>'
```

Comment:
this change is for the default behavior but causes no compatibility break as it's still possible to specify the CandidatePackages location globally instead of per environment, correct?

Reply:
correct :)

Reply:
LGTM then